Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalCheckoutRequest.h

```diff
@@ -50,13 +50,14 @@ typedef NS_ENUM(NSInteger, BTPayPalRequestIntent) {
 @property (nonatomic, readonly, strong) NSString *amount;
 
 /**
- Optional: A valid ISO currency code to use for the transaction. Defaults to merchant currency code if not set.
- @note This is only used for one-time payments.
+ Optional: A three-character ISO-4217 ISO currency code to use for the transaction. Defaults to merchant currency code if not set.
+
+ @note See https://developer.paypal.com/docs/api/reference/currency-codes/ for a list of supported currency codes.
 */
 @property (nonatomic, nullable, copy) NSString *currencyCode;
 
 /**
- Optional: Payment intent. Only applies when using checkout flow. Defaults to `BTPayPalRequestIntentAuthorize`.
+ Optional: Payment intent. Defaults to BTPayPalRequestIntentAuthorize. Only applies to PayPal Checkout.
```

Comment:
```suggestion
 Optional: Payment intent. Defaults to BTPayPalRequestIntentAuthorize.
```
A bit of a nit pick, but now that we have separate subclasses maybe this is implied?

Reply:
Totally, I went back and forth about this. The reason I left it in is because it might be helpful to someone who right-clicks on a field to see the docs in Xcode, since it might not be immediately apparent whether it's defined on `BTPayPalCheckoutRequest` or `BTPayPalRequest`. I also left it in for our team's sake since this is a new change and I want to make sure we're clear on what belongs where.